sklearn/cluster/k_means_.py

```diff
@@ -1262,6 +1264,15 @@ class MiniBatchKMeans(KMeans):
     Notes
     -----
     See http://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf
+
+    See also
+    --------
+
+    KMeans:
```

Comment:
here the notation is the same as in KMeans class. We need to decide whether we want to fix it in both places or just follow the old template (which might be misleading).

However, I must note that it rendered successfully in my local build, the link works perfectly.